docs/en/operations/access_rights.md

```diff
@@ -1,106 +1,65 @@
-# Access Rights {#access-rights}
+# Access Control and Account Management {#access-control}
 
-Users and access rights are set up in the user config. This is usually `users.xml`.
+ClickHouse support access control management based on [RBAC](https://en.wikipedia.org/wiki/Role-based_access_control) approach.
 
-Users are recorded in the `users` section. Here is a fragment of the `users.xml` file:
+Elements of ClickHouse access control model:
+- [User account](#user-account-management)
+- [Role](#role-management)
+- [Row Policy](#row-policy-management)
```

Comment:
Also quotas and settings profiles. I call such elements "access entities". If you don't mind I'd prefer to use this name.